### PR TITLE
Bug - Image downloading

### PR DESCRIPTION
**Pull request by Jandev**
Added logger to the CfpController for logmessages.
Using the appsetting in code which is also used in the templates (causation for errors)
Specifying the storage account in the Function in order to save an image to the specified account.

I'm not receiving any AppInsights messages locally, but seen this work once after a right-click deployment, so I figure it should also work after an ARM-template deployment.

### CODE REVIEW

File: Deployment/complete-solution.json [hunk 1]
```diff
@@ -561,6 +564,7 @@
             "FUNCTIONS_EXTENSION_VERSION": "~1",
             "ServicebusQueueConnectionString": "[parameters('servicebusManageConnectionString')]",
```
Comment (staal-it) [inline Deployment/complete-solution.json]: Shouldn't this be 'ServicebusEventImagesQueueConnectionString'?

Comment (Jandev) [reply Deployment/complete-solution.json]: Yes, that would be the most obvious indeed.
However, there is a small nuance between the two. The one used in the web application needs to have the queue specified in the connectionstring. The one in the Function doesn't need this as it's specified in the attribute itself.
Quite confusing, I'll check if I can fix some of this in the template, or readme.

**Comment by Jandev** [comment]
Ah yes, forgot to mention this in the PR. Good thing you noticed this before merging :-)

File: Deployment/complete-solution.json (hunk 1 above)
Comment (Jandev) [reply Deployment/complete-solution.json]: For completeness, the other comment was written on my phone, the connectionstring in the Function looks like this `Endpoint=sb://[myServicebus].servicebus.windows.net/;SharedAccessKeyName=FunctionsAccessKey;SharedAccessKey=[theMainPrimaryKey]`

The connectionstring in the App Service looks like this: `Endpoint=sb://[myServicebus].servicebus.windows.net/;SharedAccessKeyName=SendEventImages;SharedAccessKey=[aDifferentAccessKeyDueToTargettingADifferentEntityInAzure];EntityPath=eventimages`

Hope this clears it up a bit :-)

Comment (jfversluis) [reply Deployment/complete-solution.json]: Check! Those were already there! Now I just have to get #89 right and I can start deploying again

**Pull request merged**